thoth/metrics_exporter/jobs.py

```diff
@@ -171,6 +171,41 @@ def get_ceph_results_per_type():
             _LOGGER.exception(excptn)
 
 
+def get_inspection_results_per_identifier():
+    """Get the total number of inspections in Ceph per identifier."""
+    store = InspectionResultsStore()
+    try:
+        if not store.is_connected():
+            store.connect()
+        all_document_ids = store.get_document_listing()
+
+        list_ids = [str(cid) for cid in all_document_ids]
+        specfic_list_ids = {}
+        specfic_list_ids["without_identifier"] = []
+        for ids in list_ids:
+            flat_ids = ids.split("-")[1:(len(ids.split("-")) - 1)]
+            if flat_ids:
+                inspection_filter = "_".join(flat_ids)
+                if inspection_filter not in specfic_list_ids.keys():
+                    specfic_list_ids[inspection_filter] = []
+                    specfic_list_ids[inspection_filter].append(ids)
+                else:
+                    specfic_list_ids[inspection_filter].append(ids)
+            else:
+                specfic_list_ids["without_identifier"].append(ids)
+
+        for identifier, identifier_list in specfic_list_ids.items():
+            metrics.inspection_results_ceph.labels(identifier).set(len(identifier_list))
+            _LOGGER.debug(f"inspection_results_ceph for {identifier} ={len(identifier_list)}")
```

Comment:
This might be more memory efficient as we do not keep them, but rather iterate over them lazily:

```python
        specfic_list_ids = {
            "without_identifier": 0
        }
        for ids in store.get_document_listing():
            flat_ids = ids.split("-")[1:(len(ids.split("-")) - 1)]
            if flat_ids:
                inspection_filter = "_".join(flat_ids)
                specfic_list_ids[inspection_filter] += specfic_list_ids.get(inspection_filter, 0) + 1
            else:
                specfic_list_ids["without_identifier"] += specfic_list_ids["without_identifier"]
```